Caboodle/Preferences/Preferences.android.cs

```diff
@@ -121,6 +118,15 @@ T Get<T>(string key, T defaultValue)
 						case float f:
 							value = sharedPreferences.GetFloat(key, f);
 							break;
+						case string s:
+							// the case when the string is not null
+							value = sharedPreferences.GetString(key, s);
+							break;
+						default:
+							// the case when the string is null
+							if (typeof(T) == typeof(string))

```

Comment:
If we pass in null for a string we should throw an exception, as they aren't supported.

Reply:
This is just the get. The user may what to get a string, and return null if there is no value.

Reply:
ahhh gotcha